### PR TITLE
Add Inso CLI on Docker doc

### PR DESCRIPTION
**Pull request by filfreire**
### Summary

Adds information about [Inso CLI on Docker](https://hub.docker.com/r/kong/inso), available in beta, and will be introduced in `2022.4.0` GA.

### Testing

- Clone insomnia-docs repo
- Run `make setup run`
- Visit `localhost:4000`

Screenshot 2022-06-06 at 18 47 51

### CODE REVIEW

**Comment by wdawson** [comment]
Looks great! Thanks!

**Review by wongstein — APPROVED** [review]
Well done!

File: docs/inso-cli/inso-on-docker.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: article-detail
+title:  Inso CLI on Docker
+category: "Inso CLI"
+category-url: inso-cli
+---
+
+## Pull `kong/inso`
+
+To pull the latest `kong/inso` Docker image:
```
Comment (lena-larionova) [inline docs/inso-cli/inso-on-docker.md:10]:
```suggestion
Pull the latest `kong/inso` Docker image:
```

File: docs/inso-cli/inso-on-docker.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+```shell
+docker pull kong/inso:latest
+```
+
+All available tags can be found on Inso-CLI's [Docker Hub page](https://hub.docker.com/r/kong/inso/tags).
+
+## Run Inso CLI commands
+
+To run Insomnia specs in `kong/inso` container, mount the specs folder on your host machine to a `/var/temp` folder on the container. Examples:
```
Comment (lena-larionova) [inline docs/inso-cli/inso-on-docker.md:20]:
```suggestion
To run Insomnia specs in `kong/inso` container, mount the specs folder on your host machine to a `/var/temp` folder in the container. See the following sections for some examples.
```

File: docs/inso-cli/inso-on-docker.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+- Mount an Insomnia git sync repository folder to a folder on the container:
```
Comment (lena-larionova) [inline docs/inso-cli/inso-on-docker.md:22]:
```suggestion
## Examples

### Mount git sync repo

Mount an Insomnia git sync repository folder to a folder in the container:
```

File: docs/inso-cli/inso-on-docker.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+cd <your-git-sync-repo-folder>
+
+# Run Unit Tests
+docker run -it --rm -v $(pwd):/var/temp kong/inso:latest run test -w /var/temp
+
+# Generate Kong Declarative config
+docker run -it --rm -v $(pwd):/var/temp kong/inso:latest generate config -w /var/temp -f json
+```
+
+- Mount the Insomnia Application Data folder:
```
Comment (lena-larionova) [inline docs/inso-cli/inso-on-docker.md:34]:
```suggestion
## Mount Application Data folder 

Mount the Insomnia Application Data folder:
```

File: docs/inso-cli/inso-on-docker.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+```shell
+# On macOS:
+docker run -v $HOME/Library/Application\ Support/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp
+# On Linux:
+docker run -v $HOME/.config/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp
+# On Windows (using Docker for Windows and WSL):
+docker run -v /mnt/c/Users/<your_username>/AppData/Roaming/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp
```
Comment (lena-larionova) [inline docs/inso-cli/inso-on-docker.md:42]:
```suggestion
# On macOS:
docker run -v $HOME/Library/Application\ Support/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp

# On Linux:
docker run -v $HOME/.config/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp

# On Windows (using Docker for Windows and WSL):
docker run -v /mnt/c/Users/<your_username>/AppData/Roaming/Insomnia:/var/temp -it --rm kong/inso:latest run test --src /var/temp
```

File: docs/inso-cli/inso-on-docker.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+- Mount the folder where you keep an Insomnia v4 export:
```
Comment (lena-larionova) [inline docs/inso-cli/inso-on-docker.md:45]:
```suggestion
### Mount v4 export folder

Mount the folder where you keep an Insomnia v4 export:
```

**Comment by filfreire** [comment]
@lena-larionova, I've went ahead and committed your suggestions 👍

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**